services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -43,8 +47,21 @@ def _to_names(enum_cls, names):
 
 
 def validate_user_status(*, user: dict, support_email: str):
+    """
+
+    Raises:
+        web.HTTPUnauthorized
+    """
+    assert "role" in user  # nosec
+
     user_status: str = user["status"]
 
+    if user_status == DELETED:
+        raise web.HTTPUnauthorized(
+            reason=MSG_USER_DELETED.format(support_email=support_email),
+            content_type=MIMETYPE_APPLICATION_JSON,
+        )  # 401
```

Comment:
what does this comment mean 401? I think it can be removed since it does not say much, the web.HTTPUnauthorized is enough in my option